var/spack/repos/builtin/packages/r-remotes/package.py

```diff
@@ -23,5 +23,3 @@ class RRemotes(RPackage):
     version('2.1.0', sha256='8944c8f6fc9f0cd0ca04d6cf1221b716eee08facef9f4b4c4d91d0346d6d68a7')
 
     depends_on('r@3.0.0:', type=('build', 'run'))
-    depends_on('subversion', type='run')
-    depends_on('git', type='run')
```

Comment:
These are listed under the `SystemRequirements:` field so should not be removed.